recipe/meta.yaml

```
@@ -1,78 +1,79 @@
 {% set name = "onnxruntime" %}
-{% set version = "1.17.1" %}
+{% set version = "1.20.1" %}
 
 package:
   name: {{ name|lower }}{{ suffix }}
   version: {{ version }}
 
 source:
   - url: https://github.com/microsoft/onnxruntime/archive/refs/tags/v{{ version }}.tar.gz
-    sha256: 88335a1ccbccccb35d5d3aaadee090a9745cf90d97ca87c4a3ac5c4b694b7b17
+    sha256: d4c005506a2bbf88a838b14f8d1578406b8be2fb64abb50beeff908fb272529e
     patches:
-      - patches/never_call_install_python_deps.patch
+      - patches/dont-call-pip-on-win.patch                                  # [win]
       # Workaround for https://github.com/conda-forge/onnxruntime-feedstock/pull/56#issuecomment-1586080419
       - patches/windows_workaround_conflict_onnxruntime_dll_system32.patch  # [win]
-      - patches/skip_onnx_dependent_tests.patch
-      - patches/skip_moe_test.patch                                                # [(ep_variant == "cuda")]
-      - patches/skip_custom_op_test_with_cuda.patch                                # [(ep_variant == "cuda")]
-      - patches/fix_cwd_name_on_win.patch                                          # [win]
+      - patches/0001-remove-broken-cuda-moe-test-case.patch                 # [(ep_variant == "cuda")]
+      - patches/0002-skip-testing-python-iobinding.patch                    # [(ep_variant == "cuda")]
+      - patches/skip_custom_op_test_with_cuda.patch                         # [(ep_variant == "cuda")]
+      - patches/fix_cwd_name_on_win.patch                                   # [win]
 
 build:
   number: 0
   skip: True # [ppc64le or s390x]
```

```suggestion
  skip: True # [s390x]
```
We do not support `ppc64le` anymore